tasks.yaml

```diff
@@ -84,13 +85,17 @@ tasks:
       - task: setup:k3d-test-cluster
       - task: deploy:test-bundle
       - task: test
+      - task: compliance:validate
 
   - name: test-upgrade
     description: Test an upgrade from the latest released package to the current branch
     actions:
       - task: create-podinfo-latest-release-bundle
       - task: setup:k3d-test-cluster
       - task: deploy:test-bundle
+      - task: compliance:validate
```

Comment:
we prolly don't need this here since it is just testing the previously released stuff again

Reply:
Great callout - we're including this for the following reason:

> The test-upgrade task executes two validations - setting the minimum results required for evaluate to execute.

This allows us to test the `evaluate` command which expects 2 or more results for comparison. The recommendation outside of this repository is to include the assessment-results as a threshold - but we did not see the need to include that here as an item that requires maintenance.

Reply:
ah ok cool I missed that part.